jwst/fits_generator/verifiers.py

```diff
@@ -124,8 +124,7 @@ def in_range(val, min=0, max=0, inclusive=True):
     assert isinstance(min, (int, float))
     assert isinstance(max, (int, float))
 
-    if ((inclusive and val < min or val > max) or
-        (val <= min or val >= max)):
+    if ((inclusive and val < min or val > max) or (val <= min or val >= max)):
```

Comment:
This appears to be another bug: `(inclusive and val < min or val > max)`. I think it should have been: `inclusive and (val < min or val > max)`. Like this:

```suggestion
    if (inclusive and not min <= val <= max) or not min < val < max:
```